### PR TITLE
🌐 [i18n-KO] Translated `tasks/monocular_depth_estimation.mdx` to Korean

### PR DESCRIPTION
**Pull request by HanNayeoniee**
# What does this PR do?

Translated the `tasks/monocular_depth_estimation.mdx` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

Team PseudoLab, may you please review this PR? @0525hhgus, @KIHOON71, @sim-so, @gabrielwithappy, @HanNayeoniee, @wonhyeongseo, @jungnerd

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@sgugger, @ArthurZucker, @eunseojo May you please review this PR?

### CODE REVIEW

File: docs/source/ko/tasks/monocular_depth_estimation.mdx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# 단일 영상 기반 깊이 추정[[depth-estimation-pipeline]]
+
+단일 영상 기반 깊이 추정은 한 장면의 단일 이미지에서 장면의 깊이 정보를 예측하는 컴퓨터 비전 작업입니다.
+즉, 단일 카메라 시점의 장면에 있는 물체의 거리를 예측하는 과정입니다.
+
+단일 영상 기반 깊이 추정에는 3D 재구성, 증강 현실, 자율 주행, 로봇 공학 등 다양한 분야에서 응용됩니다.
```
Comment (sim-so) [inline docs/source/ko/tasks/monocular_depth_estimation.mdx:18]:
```suggestion
단일 영상 기반 깊이 추정은 3D 재구성, 증강 현실, 자율 주행, 로봇 공학 등 다양한 분야에서 응용됩니다. 
```

File: docs/source/ko/tasks/monocular_depth_estimation.mdx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+시작하기 전에, 필요한 모든 라이브러리가 설치되어 있는지 확인하세요:
+
+```bash
+pip install -q transformers
+```
+
+## 깊이 추정 파이프라인[[depth-estimation-inference-by-hand]]
+
+깊이 추정을 추론하는 가장 간단한 방법은 해당 기능을 제공하는 [`pipeline`]을 사용하는 것입니다.
+[Hugging Face Hub 체크포인트](https://huggingface.co/models?pipeline_tag=depth-estimation&sort=downloads) 에서 파이프라인을 초기화합니다:
```
Comment (sim-so) [inline docs/source/ko/tasks/monocular_depth_estimation.mdx:47]:
```suggestion
[Hugging Face Hub 체크포인트](https://huggingface.co/models?pipeline_tag=depth-estimation&sort=downloads)에서 파이프라인을 초기화합니다:
```

File: docs/source/ko/tasks/monocular_depth_estimation.mdx [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+```
+
+<div class="flex justify-center">
+     <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/transformers/tasks/depth-visualization.png" alt="Depth estimation visualization"/>
+</div>
+
+## 직접 깊이 추정 추론하기
+
+이제 깊이 추정 파이프라인 사용법을 살펴보았으니 동일한 결과를 복제하는 방법을 살펴보겠습니다.
+[Hugging Face Hub 체크포인트](https://huggingface.co/models?pipeline_tag=depth-estimation&sort=downloads)에서 모델과 관련 프로세서를 로드하는 것부터 시작합니다.
```
Comment (sim-so) [inline docs/source/ko/tasks/monocular_depth_estimation.mdx:95]:
```suggestion
[Hugging Face Hub 체크포인트](https://huggingface.co/models?pipeline_tag=depth-estimation&sort=downloads)에서 모델과 관련 프로세서를 가져오는 것부터 시작합니다.
```

File: docs/source/ko/tasks/monocular_depth_estimation.mdx [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+```py
+>>> from transformers import AutoImageProcessor, AutoModelForDepthEstimation
+
+>>> checkpoint = "vinvino02/glpn-nyu"
+
+>>> image_processor = AutoImageProcessor.from_pretrained(checkpoint)
+>>> model = AutoModelForDepthEstimation.from_pretrained(checkpoint)
+```
+
+필요한 이미지 변환을 처리하는 'image_processor'를 사용하여 모델에 대한 이미지 입력을 준비합니다.
```
Comment (sim-so) [inline docs/source/ko/tasks/monocular_depth_estimation.mdx:107]:
```suggestion
필요한 이미지 변환을 처리하는 `image_processor`를 사용하여 모델에 대한 이미지 입력을 준비합니다.
```

File: docs/source/ko/tasks/monocular_depth_estimation.mdx [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+'image_processor'는 크기 조정 및 정규화 등 필요한 이미지 변환을 처리합니다:
```
Comment (sim-so) [inline docs/source/ko/tasks/monocular_depth_estimation.mdx:108]:
```suggestion
`image_processor`는 크기 조정 및 정규화 등 필요한 이미지 변환을 처리합니다:
```

**Review by sim-so — CHANGES_REQUESTED** [review]
관심 있던 주제의 문서를 읽을 수 있어서 좋았습니다!
고생 많으셨습니다😄

**Review by sim-so — CHANGES_REQUESTED** [review]
관심 있던 주제의 문서를 읽을 수 있어서 좋았습니다!
고생 많으셨습니다😄

**Review by sim-so — CHANGES_REQUESTED** [review]
관심 있던 주제의 문서를 읽을 수 있어서 좋았습니다!
고생 많으셨습니다😄

File: docs/source/ko/tasks/monocular_depth_estimation.mdx [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+
+```py
+>>> predictions["depth"]
+```
+
+<div class="flex justify-center">
+     <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/transformers/tasks/depth-visualization.png" alt="Depth estimation visualization"/>
+</div>
+
+## 직접 깊이 추정 추론하기
```
Comment (0525hhgus) [inline docs/source/ko/tasks/monocular_depth_estimation.mdx:92]:
```suggestion
## 직접 깊이 추정 추론하기[[depth-estimation-inference-by-hand]]
```
TOC를 추가했습니다!

**Review by 0525hhgus — CHANGES_REQUESTED** [review]
좋은 번역 감사합니다 😀 
저는 다른 피드백 없이 toc 부분만 리뷰 남겼습니다 👍

**Review by ArthurZucker — APPROVED** [review]
Thanks for keeping up with the effort! 
LGTM!

**Pull request merged**